client/components/backup-retention-management/info-tooltip/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Button, Gridicon } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import * as React from 'react';
+import Tooltip from 'calypso/components/tooltip';
+import { STORAGE_RETENTION_LEARN_MORE_LINK } from '../constants';
+import './style.scss';
+
+const InfoToolTip: React.FC = () => {
+	const translate = useTranslate();
+	const [ isTooltipVisible, setTooltipVisible ] = React.useState< boolean >( false );
+	const tooltip = React.useRef< SVGSVGElement >( null );
+
+	const toggleTooltip = ( event: React.MouseEvent< HTMLButtonElement, MouseEvent > ): void => {
+		setTooltipVisible( ! isTooltipVisible );
+		// when the info tooltip inside a button, we don't want clicking it to propagate up
+		event.stopPropagation();
+	};
+
+	const closeTooltip = () => {
+		setTooltipVisible( false );
+	};
+
+	return (
+		<div className="retention-setting-info-tooltip">
+			<Button
+				borderless
+				className="retention-setting-info-tooltip__toggle-tooltip"
+				onClick={ toggleTooltip }
+			>
+				<Gridicon ref={ tooltip } icon="info-outline" />
+			</Button>
+			<Tooltip
+				className="retention-setting-info-tooltip__tooltip"
+				isVisible={ isTooltipVisible }
+				position="right"
+				context={ tooltip.current }
+				showOnMobile
+			>
+				<p>
+					{ translate(
+						'You can manage the storage used by changing how many days of backups will be saved.'
+					) }
+					<Button
+						borderless
+						compact
+						className="retention-setting-info-tooltip__close-tooltip"
+						onClick={ closeTooltip }
+					>
+						<Gridicon icon="cross" size={ 18 } />
+					</Button>
+				</p>
+				<hr />
```

Comment:
Using `hr`s is a suboptimal way to lay out the presentation. Could be a better bet to add an extra wrapper and use CSS.